features/_your_results/limit_tax_relief.feature

```diff
@@ -0,0 +1,68 @@
+Feature:
+  As an employee contributing greater than £40,000 per year to my pension
+  In order to get a clear picture of the impact on my take-home pay.
```

Comment:
Sorry, this is a minor one. The order of convention I've used across others has been `In order..., As a..., I want...`
Could you maybe switch lines 2 and 3 to follow this?